lib/Dialect/SV/SVOps.cpp

```diff
@@ -1055,6 +1057,153 @@ LogicalResult PAssignOp::verify() {
   return success();
 }
 
+// Return true if lhs + offset = rhs.
+static bool isOffset(Value lhs, uint64_t offset, Value rhs) {
```

Comment:
This method is already available in `HWOps.cpp`. Could it be exposed in `HWOps.h` and included here to avoid duplication?

Reply:
Thanks, created a helper function in HWOps.h